src/windows/region.py

```diff
@@ -119,8 +119,13 @@ def __init__(self, file=None, clip=None):
         self.videoPreview.setSizePolicy(QSizePolicy.Preferred, QSizePolicy.Expanding)
         self.verticalLayout.insertWidget(0, self.videoPreview)
 
+        # Set aspect ratio to match source content
+        aspect_ratio = openshot.Fraction(self.width, self.height)
+        aspect_ratio.Reduce()
+        self.videoPreview.aspect_ratio = aspect_ratio
+
         # Set max size of video preview (for speed)
-        self.viewport_rect = self.videoPreview.centeredViewport(self.videoPreview.width(), self.videoPreview.height())
+        self.viewport_rect = self.videoPreview.centeredViewport(self.width, self.height)
```

Comment:
Does this hurt performance? Or is this only used when processing effects, etc..?